### PR TITLE
[feat] change visual to ppocr

### PR DESCRIPTION
**Pull request by heepengpeng**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

modify visual 

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: deploy/py_infer/src/utils/visual_utils.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+OCR visualization methods
+"""
+import math
+import os.path
+import random
+
+import cv2
+import numpy as np
+from PIL import Image, ImageDraw, ImageFont
+
```
Comment (liangxhao) [inline deploy/py_infer/src/utils/visual_utils.py:11]: 添加__all__，定义那些函数是可以被外部调用的

File: deploy/py_infer/src/utils/visual_utils.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+def vis_bbox(image, box_list, color, thickness):
+    image = image.copy()
```
Comment (liangxhao) [inline deploy/py_infer/src/utils/visual_utils.py:14]: 每个函数添加些接口注释

File: deploy/py_infer/src/utils/visual_utils.py (hunk 1 above)
Comment (heepengpeng) [reply deploy/py_infer/src/utils/visual_utils.py:11]: 已修改

File: deploy/py_infer/src/utils/visual_utils.py (hunk 2 above)
Comment (heepengpeng) [reply deploy/py_infer/src/utils/visual_utils.py:25]: 已修改

File: docs/cn/inference/inference_tutorial.md [hunk 3]
```diff
@@ -75,13 +85,18 @@ python infer.py \
...
     --det_model_name_or_config=../../configs/det/dbnet/db_r50_icdar15.yaml \
-    --res_save_dir=det
+    --res_save_dir=det \
+    --vis_det_save_dir=det
 ```
 
-结果保存在det/det_results.txt，格式如下：
+可视化结果存放在det中，如图所示：
+
+![det](https://github.com/heepengpeng/mindocr/assets/122354463/e7e708f7-7a69-4f3f-9234-bad909052e7a)
```
Comment (liangxhao) [inline docs/cn/inference/inference_tutorial.md:94]: 给每张图片添加一些html标记，控制显示图片的大小

Comment (heepengpeng) [reply docs/cn/inference/inference_tutorial.md]: 已修改

**Review by liangxhao — APPROVED** [review]

**Review by jianyunchao — APPROVED** [review]

**Pull request merged**